File: src/main/java/de/florianschmitt/service/VolunteerService.java
package de.florianschmitt.service;

import de.florianschmitt.model.entities.EVolunteer;
import de.florianschmitt.repository.VolunteerRepository;
import org.springframework.stereotype.Service;

@Service
public class VolunteerService extends AbstractPageableAdminService<EVolunteer, VolunteerRepository> {
}

File: src/main/kotlin/de/florianschmitt/service/VolunteerService.kt
package de.florianschmitt.service

import de.florianschmitt.model.entities.EVolunteer
import de.florianschmitt.repository.VolunteerRepository
import de.florianschmitt.service.base.AbstractPageableAdminService
import org.springframework.stereotype.Service

@Service
class VolunteerService : AbstractPageableAdminService<EVolunteer, VolunteerRepository>()
